nav2_system_tests/CMakeLists.txt

```diff
@@ -47,6 +47,40 @@ set(dependencies
   behaviortree_cpp_v3
 )
 
+set(local_controller_plugin_lib local_controller)
```

Comment:
This can be placed into the CMakeLists file in the src/error_codes directory that we can include `add_subdirectory(src/error_codes)`

Reply:
I tried this, and the servers were unable to find the plugins.

Reply:
Was this all specified before the tests themselves?

Reply:
It can't be inside the `if(BUILD_TESTING)`.

Reply:
That's probably why.

Reply:
oh, sorry I misunderstood. 

Yes, it was placed before the tests. I though you were referring to the first cmake file.